src/components/domains/storyPost/DisplayStoryPostPaper/DisplayStoryPostPaper.tsx

```diff
@@ -85,19 +88,33 @@ export const DisplayStoryPostPaper: VFC<Props> = ({
   const handleClickEmoji = useCallback(
     async (emojiId: string) => {
       try {
-        await restClient.apiPost<Reaction>('/reactions', {
+        if (!storyPost.currentUserReaction?._id) {
```

Comment:
currentUserReaction が存在しているとき _id は必ず存在しているので

```suggestion
        if (!storyPost.currentUserReaction) {
```
で良いかも